src/Symfony/Component/Serializer/Mapping/Loader/YamlFileLoader.php

```diff
@@ -103,4 +84,38 @@ public function loadClassMetadata(ClassMetadataInterface $classMetadata)
 
         return false;
     }
+
+    /**
+     * Return the names of the classes mapped in this file.
+     *
+     * @return string[] The classes names
+     */
+    public function getMappedClasses()
+    {
+        return array_keys($this->loadClassesFromYaml());
+    }
+
+    private function loadClassesFromYaml()
+    {
+        if (!stream_is_local($this->file)) {
```

Comment:
Should be stored in ``$this->classes`` in this method to benefit to ``getMappedClasses``.